client/components/public/FAQ.jsx

```diff
@@ -542,10 +541,39 @@ FAQ = class FAQ extends Component {
             </Grid>
           </Accordion.Content>
           
+          <Accordion.Title active={activeIndex === 18} index={18} onClick={(e,p) => this.handleClick(e,p)} >
+            <Icon color="purple" name="dropdown"/>
+            <Icon name="dollar"/>
+            How can I support the Great Puzzle Hunt?
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 18}>
+            <p> We keep this event free to you thanks to generous donations from our sponsors, and from gear purchases. These contributions help cover costs of this event including web fees and development, graphics, prizes, advertising, and much more. </p>
+            <p>You can help out by:
+              <ul>
+                <li> Donating any amount. $5 and $10 donations make a difference. </li>
+                 <li> Purchasing a shirt!</li>
+		<li> Volunteering!* It takes a team to make the Puzzle Hunt happen, and we are grateful for everyone who helps out.</li>
+	      </ul>
+	    </p> 
+            <p> The WWU Great Puzzle Hunt operates under WWU Foundation's 501(c)(3) status, so all donations are tax deductible. </p>
+            <p> <strong>*</strong> Volunteers choose the "volunteer" account type when <a href="/register">registering</a>. Questions? Contact <a href="mailto:info@greatpuzzlehunt.com">info@greatpuzzlehunt.com</a>. </p>
```

Comment:
Do we generally attract volunteers this way? For some reason, I thought most folks filtered through Millie and then she directed them to create an account on the site. I'm not sure what the communication path looks like if they start on the site. I guess we'll just reach out to them at some point before the event? I definitely like the inclusion of this item!